### DISCUSSION TITLE
Dropdown with search capabilites

### DISCUSSION THREAD

**Discussion by razvanilin** [heroui-inc/heroui#771] [Help] [answered]
Hey everyone,

I was wondering if anyone encountered this use case before. Have a dropdown with an `Input` trigger that allows you to type text and filter through the dropdown items. Check the examples below.

I managed to do it (sort of), but I'm not 100% happy with the behavior. When the input is clicked, the dropdown opens and the focus is taken away from the input box. When the input box is clicked so the user can type, the dropdown closes. Also, after the user presses the `enter` key, the dropdown appears.

Does anyone know of a way to keep the dropdown open while the input is in focus?

Code: https://codesandbox.io/s/modern-bash-knnz6t?file=/src/App.js

![image](https://user-images.githubusercontent.com/5342321/190843650-8e1f316c-b3f9-40da-a225-9d3fa4206a2b.png)

**Comment by marcosfreitas** [comment]
I was expecting that the current select component could work with search. How did you achieve that?

**Reply by luiskabes-arch** [reply, marked as answer]
@marcosfreitas you can use [Autocomplete](https://nextui.org/docs/components/autocomplete) component for that

**Reply by razvanilin** [reply]
Yup, Autocomplete was not a thing when I made this post, but now I use that component for my use case. I do have a component where I want a more of a Dropdown behaviour and for that I use the Popover component with a filter Input and ListBox that shows only the items that match the filter

**Reply by luiskabes-arch** [reply]
@razvanilin I understand, since your issue was published 2 years ago, anyway you should mark this issue to be answered for help another people which facing the same issue
